### PR TITLE
Muharrem - My Photos Kotlin Version

### PR DESCRIPTION
**Pull request by Muharrem-sdet**
Week 8 Day 6 My Photos Kotlin Version with Retrofit

### CODE REVIEW

File: MyPhotoKotlinVersion/app/src/main/java/com/example/myphotoskotlinversion/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+class MainActivity : AppCompatActivity() {
+
+    lateinit var recyclerView: RecyclerView
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        recyclerView = findViewById(R.id.recycler_View)
```
Comment (oliver-basoglu) [inline MyPhotoKotlinVersion/app/src/main/java/com/example/myphotoskotlinversion/MainActivity.kt:24]: in XML named it as photos_recycler_view

File: MyPhotoKotlinVersion/app/src/main/java/com/example/myphotoskotlinversion/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            override fun onFailure(call: Call<MutableList<Photo>>, t: Throwable) {
+                t.stackTrace
+            }
+
+            override fun onResponse(
+                call: Call<MutableList<Photo>>,
+                response: Response<MutableList<Photo>>
+            ) {
+             val photoList = response.body()
+                if (photoList != null) {
```
Comment (oliver-basoglu) [inline MyPhotoKotlinVersion/app/src/main/java/com/example/myphotoskotlinversion/MainActivity.kt:38]: try to use let

**Comment by dzemal5** [comment]
amazing guys, keep up the good work!

**Review by dzemal5 — CHANGES_REQUESTED** [review]
make sure you correct your codes based on the instructor s recommendation.

File: MyPhotoKotlinVersion/app/src/main/java/com/example/myphotoskotlinversion/model/Photo.kt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+import java.io.Serializable
+
+data class Photo(
+    val albumID: Int,
+    val id: Int,
+    val title: String,
+    val url: String,
+    val thumbnailURL: String
+): Serializable
```
Comment (oliver-basoglu) [inline MyPhotoKotlinVersion/app/src/main/java/com/example/myphotoskotlinversion/model/Photo.kt:11]: try to use Parceable

**Pull request merged**